### PR TITLE
feat: add scripts for common k8s tasks

### PR DESCRIPTION
**Pull request by tklovett**

### CODE REVIEW

File: bin/k-secret [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
```
Comment (knksmith57) [inline bin/k-secret:2]:
```suggestion
set -eo pipefail
```

File: bin/k-secret [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+name="$1"
+subpath="$2"
+
+secret_as_json=$(kubectl get --output json secret "$name")
+
+if [[ "$subpath" = "" ]]; then
```
Comment (knksmith57) [inline bin/k-secret:10]:
```suggestion
if [[ -z "$subpath" ]]; then
```

File: bin/k-secret [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+name="$1"
+subpath="$2"
+
+secret_as_json=$(kubectl get --output json secret "$name")
```
Comment (knksmith57) [inline bin/k-secret]:
```suggestion
secret_as_json="$(kubectl get --output json secret "$name")"
```

File: bin/k-secret [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+if [[ "$subpath" = "" ]]; then
+  echo "$secret_as_json" | jq '.data | to_entries[].key'
```
Comment (knksmith57) [inline bin/k-secret]:
```suggestion
  echo "$secret_as_json" | jq '.data | keys[]'
```

File: bin/k-secret (hunk 2 above)
Comment (tklovett) [reply bin/k-secret:10]: The way I have it is a Tannerism -- an equality comparison is easier to remember and understand than the mysterious "`-z`"

**Pull request merged**